public/app/features/explore/NodeGraphContainer.tsx

```diff
@@ -19,19 +19,23 @@ export function UnconnectedNodeGraphContainer(props: Props & ConnectedProps<type
   const { dataFrames, range, splitOpen, short } = props;
   const getLinks = useLinks(range, splitOpen);
 
+  const [open, setOpen] = useState(true);
+
   return (
-    <div style={{ height: short ? 300 : 600 }}>
-      <Collapse
-        label={
-          <span>
-            Node graph <Badge text={'Beta'} color={'blue'} icon={'rocket'} tooltip={'This visualization is in beta'} />
-          </span>
-        }
-        isOpen
-      >
+    <Collapse
+      label={
+        <span>
+          Node graph <Badge text={'Beta'} color={'blue'} icon={'rocket'} tooltip={'This visualization is in beta'} />
+        </span>
+      }
+      collapsible={short}
+      isOpen={short ? open : true}
```

Comment:
At this point the collapsing is allowed only when shown together with trace view